src/codes/june_challenge/day3/TwoCityScheduling.cpp

```diff
@@ -0,0 +1,28 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+class Solution {
+public:
+    static bool sortByDiff(pair<pair<int, int>, int> &a, pair<pair<int, int>, int> &b) {
+        return a.second < b.second;
+    }
+    
+    int twoCitySchedCost(vector<vector<int>>& costs) {
+        int n = costs.size();
+        vector<pair<pair<int, int>, int>> vect;
+        for (int i = 0; i < n; i++) {
+            int diff = costs[i][0] - costs[i][1];
+            vect.push_back({{costs[i][0], costs[i][1]}, diff});
+        }
```

Comment:
You can sort the `costs` array without creating a temp vector

Reply:
Maybe use comparator directly with lambda?

Reply:
Seems like a nice idea, will do it.